### ISSUE TITLE
Feature request: Trusted Device

### ISSUE THREAD

**Issue by trymeouteh** [localsend/localsend#861]
### Is your feature request related to a problem? Please describe.

A list for devices that will allow Quick Save

### Describe the solution you'd like

A setting in a favorite device which will allow you to automatically accept file transfers from this device (Quick Save)

### Describe alternatives you've considered

None

**Comment by sergd88** [comment]
Yes, such implementation makes sense, especially considering that now there's an option to add devices to the "Favorites" category. It would be something similar to how it's done in Google's Nearby Share. Though in that case, it relates to displaying the device on the network, whereas in our case, it's about Quick Save.

Then, instead of a toggle switch in the settings, we would have a dropdown list for Quick Save with the following options:

1. Disabled;
2. Enabled only for Favorite devices;
3. Enabled for all.

In combination with a excellent feature request [#860](https://github.com/localsend/localsend/issues/860), the "Favorite" function, in my opinion, would make much more sense, as currently, it seems not very useful.
